scripts/vidarr-build

```diff
@@ -202,5 +203,9 @@ if args.command == "deploy":
                 ok = False
             else:
                 print(f"Registered on {vidarr_url} as {name}/{args.version}")
+                registered = True
+    if not registered:
+        print("Could not find a server that wanted this workflow. :-(")
```

Comment:
This would get printed _after_ 'EVERYTHING IS AWESOME!!!', which is confusing.

Reply:
We could update this line to read "....except for this: Could not find a server that wanted this workflow. :-("

(I think "EVERYTHING IS AWESOME!!!" indicates that the tests all passed)

Reply:
Yeah, that was meant to indicate that tests passed, but it is true that everything is not awesome (I found a perfect more obscure LEGO Movie 2 reference for that).

I also move the logic around so that it will fail before it runs the tests.